plugins/provisioners/docker/cap/redhat/docker_install.rb

```diff
@@ -7,7 +7,12 @@ def self.docker_install(machine)
             machine.communicate.tap do |comm|
               comm.sudo("yum -q -y update")
               comm.sudo("yum -q -y remove docker-io* || true")
-              comm.sudo("curl -sSL https://get.docker.com/ | sh")
+              if machine.guest.capability("flavor") == :rhel_8
+                # containerd.io is not available on official yum repos
+                # install it directly from docker
+                comm.sudo("yum -y install https://download.docker.com/linux/centos/7/x86_64/stable/Packages/containerd.io-1.2.6-3.3.el7.x86_64.rpm")
+              end
+              comm.sudo("curl -fsSL https://get.docker.com/ | sh")
```

Comment:
I'm curious if it would work to just add the docker repository to the system and install the `docker-ce` package instead of just installing the single package and curl/bashing the rest of the installation. The main advantage would be that we wouldn't be referencing a specific package version which will likely change in the future.